### DISCUSSION TITLE
redis/xuebeng-chuantou-jichuan

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#182] [Announcements]
# redis/xuebeng-chuantou-jichuan

简单聊聊缓存雪崩、穿透、击穿 大家好，我是二哥呀。作为后端开发，我想缓存是大家再熟悉不过的东西了。 本文会介绍出现缓存雪崩、穿透和击穿的业务背景、解决方案和对业务可靠性处理。事先说明，最佳解决方案一定需要结合实际业务调整，不同业务的处理不完全相同 其实我在网上也看过不少关于缓存雪崩、穿透、击穿介绍，不知道是不是大家所做业务的不同，发现有不少小伙伴有以下疑问，比如： 加随机时间过期后，如果访问时间刚好就是加了随机时间后的数据，这样岂不是白加了随机时间？ 热点数据不过期，那岂不是有越来越多的脏数据？

https://tobebetterjavaer.com/redis/xuebeng-chuantou-jichuan.html

**Comment by qinple** [comment]
我之前从未听说过还有本地缓存, 涨知识了
